app/models/request/FileStoreInitiateRequest.scala

```diff
@@ -19,6 +19,8 @@ package models.request
 import play.api.libs.json.{ OFormat, Json }
 
 case class FileStoreInitiateRequest(
+  fileName: Option[String] = None,
+  mimeType: Option[String] = None,
```

Comment:
I don't think that this change is needed - the filestore will ignore it - see here https://github.com/hmrc/binding-tariff-filestore/blob/master/app/uk/gov/hmrc/bindingtarifffilestore/model/upscan/v2/FileStoreInitiateRequest.scala